osu.Game.Rulesets.Catch/Mods/CatchModAutoplay.cs

```diff
@@ -20,5 +22,12 @@ protected override Score CreateReplayScore(Beatmap<CatchHitObject> beatmap)
                 Replay = new CatchAutoGenerator(beatmap).Generate(),
             };
         }
+
+        public override void ApplyToRulesetContainer(RulesetContainer<CatchHitObject> rulesetContainer)
+        {
+            if (rulesetContainer is CatchRulesetContainer)
+                ((CatchRulesetContainer)rulesetContainer).AutoMod = true;
```

Comment:
Since this is the default, I'd like to exclude it for now if possible. It should be part of `Replay` or `ReplayFrame` and handled internally to `FramedReplayInputHandler`, rather than being set by mods.

Reply:
Oh this isn't actually the default, I misread.